docs/internal/investigations/remote-runtime-configuration-dapr/poc/dapr-poc-scenario.md

```diff
@@ -0,0 +1,143 @@
+# Dapr Proof of Concept scenario
+
+
+This document describes the scenario that will be developed to prove that [Dapr](https://dapr.io/) sidecars can be configured from a central place.
+
+## Reasons
+
+Kyma Runtimes must consume fewer resources. It can be achieved by delegating some of Kyma's responsibilities
+to Dapr. Dapr injects sidecars into selected Pods where they fulfill their given tasks, such as state management or pub-sub.
+Configuration for these sidecars can be held in one central place, from where Kyma Runtimes could fetch them and configure the
+sidecars accordingly.
+
+## Known issues
+
+Dapr sidecars need a restart to apply changes in configuration. We are still waiting for the decision on the Dapr side if
+they will stay with that approach.
+
+## PoC scenario
+
+The scenario will present two Runtimes using [Dapr State API](https://github.com/dapr/docs/blob/master/reference/api/state_api.md) to persist the state in two different Redis instances.
+
+1. Control Plane admin sets the configuration for two Runtimes.
+2. Runtime Agents from both Runtimes fetch the configuration and create the Dapr Component resources.
+3. Control Plane admin changes the configuration.
+4. Agents fetch the new configuration, apply it, and restart the Pods.
+5. Both Runtimes use the new configuration.
+
+As it's the Proof of Concept stage, we can use some temporary solutions, such as:
+- Configuration held in memory instead of a database
+- Pods being restarted by the Agent instead of a more complex solution
+
+### Initial architecture
+
+![Initial Architecture Diagram](./stage-1.svg)
+
+The image depicts the initial configuration of the components which will be deployed.
+
+Tools:
+- docker [https://www.docker.com/](https://www.docker.com/)
+- k3d v3 [https://github.com/rancher/k3d](https://github.com/rancher/k3d)
+- kubectl [https://kubernetes.io/docs/tasks/tools/install-kubectl/](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- helm 3 [https://helm.sh/](https://helm.sh/)
+- watch (`brew install watch`)
+
+Ensure you have all the tools installed and start in the `poc` directory.
+
+> NOTE: Please keep in mind that some shells automatically escape some characters when copying and pasting commands. Always check the command before running it.
```

Comment:
```suggestion
>**NOTE:** Please keep in mind that some shells automatically skip some characters when copying and pasting commands. Always check the command before running it.
```

Reply:
FIne with bold, but the characters are escaped with an escape character, not skipped.